pkg/apis/extensions/types.go

```diff
@@ -87,6 +88,18 @@ type CPUTargetUtilization struct {
 	TargetPercentage int `json:"targetPercentage"`
 }
 
+// Alpha-level support for Custom Metrics in HPA (as annotations).
+type CustomMetricUtilization struct {
+	// Custom Metric name.
+	Name string `json:"name"`
+	// Custom Metric value (average).
+	Value resource.Quantity `json:"value"`
```

Comment:
Should this be `targetValue` for similarity to `cpuUtilization.targetPercentage`.